kafka-client/src/main/java/dev/responsive/db/CassandraFactSchema.java

```diff
@@ -257,44 +265,47 @@ public BoundStatement delete(
    * Inserts data into {@code table}. Note that this will overwrite
    * any existing entry in the table with the same key.
    *
-   * @param table         the table to insert into
-   * @param partitionKey  the partitioning key
-   * @param key           the data key
-   * @param value         the data value
-   *
+   * @param table        the table to insert into
+   * @param partitionKey the partitioning key
+   * @param key          the data key
+   * @param value        the data value
+   * @param timestamp    the event time of the data
    * @return a statement that, when executed, will insert the row
-   *         matching {@code partitionKey} and {@code key} in the
-   *         {@code table} with value {@code value}
+   * matching {@code partitionKey} and {@code key} in the
+   * {@code table} with value {@code value}
    */
   @Override
   @CheckReturnValue
   public BoundStatement insert(
       final String table,
       final int partitionKey,
       final Bytes key,
-      final byte[] value
+      final byte[] value,
+      long timestamp
```

Comment:
```suggestion
      final long timestamp
```

Reply:
Don't want to leave a comment on every single one but same goes for all the below

Reply:
oops... I need to change my IntelliJ settings back after some plugin messed it up